Edit.py

```diff
@@ -0,0 +1,38 @@
+import PrintList
+
+def edit(to_Do):
+  # Print List
+  PrintList.print_list(to_Do)
+
+  X = input("[Press X if you want to exit]").upper()
+
+  if X == 'x':
+    return
+  else:
+    pass
+
+  while True:
+    task_number = input("What item would you like to edit? ")
+    
+    if task_number == "X":
```

Comment:
Does this still work if the user entered a lowercase x since we're not running `upper()` above?